examples/mqtt_temperature.eve

```diff
@@ -0,0 +1,40 @@
+
+# MQTT
+## Temperature
+From mqtt://mqtt.bitraf.no see http://iot.bitraf.no
+### Define the sensors
+```eve
+commit @shared
+  [#bitraf #temperature name: "lab" topic: "bitraf/temperature/1"]
+  [#bitraf #temperature name: "office" topic: "bitraf/temperature/2/value"]
+  [#bitraf #temperature name: "outside" topic: "bitraf/temperature/3/value"]
+```
+### Show latest sensor values
+```eve
+search @shared
+  sensor = [#bitraf #temperature name topic]
+search @mqtt
+  message = [#message #incoming topic: topic, payload]
+  sort[value: payload, per: topic] = 1
```

Comment:
what is the intent here? does the payload for each topic contain historical samples that you
want to throw away?